packages/@react-spectrum/color/src/HexColorField.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+
+function HexColorField {
+  return (
+    <div>Hex</div>
+  );
+}
+
+/**
+ * HexColorField allows users to input a color value in hex format.
+ */
+const _HexColorField = React.forwardRef(HexColorField);
```

Comment:
TODO: Ask when it's appropriate to do `React.forwardRef()` for a component

Reply:
Pretty much every component 
👍